python/core/composer/qgscomposermap.sip

```diff
@@ -223,6 +223,27 @@ class QgsComposerMap : QgsComposerItem
     /** Stores the current layer styles into style overrides. @note added in 2.8 */
     void storeCurrentLayerStyles();
 
+    /** Whether the map should follow a visibility preset. If true, the layers and layer styles
+     * will be used from given preset name (configured with setFollowVisibilityPresetName() method).
+     * This means when preset's settings are changed, the new settings are automatically
+     * picked up next time when rendering, without having to explicitly update them.
+     * At most one of the flags keepLayerSet() and followVisibilityPreset() should be enabled
+     * at any time since they are alternative approaches - if both are enabled,
```

Comment:
They are mutually exclusive in GUI, however not in API (that would need a backwards incompatible change). So the note is just to make people aware. In 3.0 we could switch to an enumeration of possible modes.